guides/developers/event-display.md

```diff
@@ -56,18 +57,27 @@ Currently the sub managers are not big enough to be divided into multiple parts.
 
 ## [`UIManager`](../../packages/phoenix-event-display/src/managers/ui-manager/index.ts)
 
-The `UIManager` is responsible for all the UI aspects of the `EventDisplay`. This includes the Phoenix menu, dat.GUI menu, stats UI etc.
+The `UIManager` is responsible for all the UI aspects of the `EventDisplay`. This includes the Phoenix menu, dat.GUI menu, stats UI, etc.
 
 It takes care of tasks like:
 
 * Adding configuration options to Phoenix UI on loading geometry or event data.
-* Setting the UI color theme to dark or light
-* Applying clipping to geometries
-* Linking UI menu options with the `EventDisplay`
+* Setting the UI color theme to dark or light.
+* Applying clipping to geometries.
+* Linking UI menu options with the `EventDisplay`.
+* Configuring different color options for event data.
 * etc.
 
+### [`PhoenixUI`](../../packages/phoenix-event-display/src/managers/ui-manager/phoenix-ui.ts)  
+
+`PhoenixUI` is used as a common interface for implementing UI menus in Phoenix.  
+
+Please see the [API docs for PhoenixUI](https://hepsoftwarefoundation.org/phoenix/api-docs/interfaces/PhoenixUI.html) to have a deeper look at the available functions.
+
 The `UIManager` is further divided into the following managers:
```

Comment:
This line is out of context now. Instead of this, we can write something like:

```
`PhoenixUI` currently has the following implementations:
```

Reply:
done